app/src/app/components/specmath-modal/modal.component.ts

```diff
@@ -26,30 +26,33 @@ enum steps {
 
 type StepOptions = {
   [key in steps]: {
-    toolTipText: string,
+    toolTipText?: string,
     nextStep?: number,
```

Comment:
Could these be type 'Steps' instead of number?

Reply:
Updated